lib/mixins/flowdock.ts

```diff
@@ -0,0 +1,21 @@
+import * as request from 'request-promise';
+import { FlowdockInboxItem } from './flowdock-types';
+
+export class FlowdockAdapter {
+    public postToInbox(item: FlowdockInboxItem) {
```

Comment:
The main reason is that currently it's just FlowDock, and in the future I'm assuming we'll support this, email, possible Front, and other external services perhaps.

So it was more futureproofing, it's easier now, but initialising a load of different services like this in the future instead of just adding the particular mixin as a new `implements` seems cleaner. If you feel strongly that we shouldn't be doing this, I'm happy to remove it. :)